### DISCUSSION TITLE
How can I disable warnings for a selection of scripts/objects?

### DISCUSSION THREAD

**Discussion by mathew-odwyer** [bscotch/stitch#220] [Q&A] [answered]
I want to use [Chatterbox](https://www.jujuadams.com/Chatterbox/#/latest/) in my game but when I import it into my GM project stitch will show hundreds of errors/warnings that need to be fixed.

I don't want to have to go through and fix them all, it would be much better for me to import the local package and simply tell Stitch to disable warnings and errors for the imported package.

**Comment by mathew-odwyer** [comment, marked as answer]
Found it:

![image](https://github.com/bscotch/stitch/assets/50978201/5b010846-dfcb-4205-b1b3-c39539461486)

**Reply by adam-coster** [reply]
I'd totally forgotten I implemented this 😂
